src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -287,6 +287,12 @@ private static void register(int id, Set<String> models, BiConsumer<Position, By
                 case 3 -> p.addAlarm(Position.ALARM_CORNERING);
             }
         });
+        register(175, fmbXXX, (p, b) -> {
+            switch (b.readUnsignedByte()) {
+                case 0 -> p.addAlarm(Position.ALARM_GEOFENCE_EXIT);
+                case 1 -> p.addAlarm(Position.ALARM_GEOFENCE_ENTER);
+            }
+        });
```

Comment:
I think it's better to do something like this:

```suggestion
        register(175, fmbXXX, (p, b) -> p.addAlarm(b.readUnsignedByte() > 0 ? Position.ALARM_GEOFENCE_ENTER : ALARM_GEOFENCE_EXIT);
```